src/libs/reportUtils.js

```diff
@@ -26,10 +26,11 @@ function getReportParticipantsTitle(logins) {
  * Check whether a report action is Attachment is not.
  *
  * @param {Object} reportMessageText report action's message as text
+ * @param {Object} reportMessageHtml report action's message as html
  * @returns {Boolean}
  */
-function isReportMessageAttachment(reportMessageText) {
-    return reportMessageText === '[Attachment]';
+function isReportMessageAttachment(reportMessageText, reportMessageHtml) {
+    return reportMessageText === '[Attachment]' && reportMessageHtml !== '[Attachment]';
```

Comment:
could put the `lodashGet` in here rather than passing it in as an argument.

Reply:
Whoops! Sorry @parasharrajat, didn't see [this](https://github.com/Expensify/App/pull/8117#pullrequestreview-908169352) before leaving this comment 😅

Reply:
No worries.